website/src/main/tut/docs/changelog.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: docs
+title: Changelog
+---
+
+# Changelog
```

Comment:
I think we can remove this section now. I kept it in the old page just to redirect old links. Now we already break all links so there's no point in keeping this section.

Reply:
👍